improver_tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -474,6 +474,56 @@ def test_without_weights(self):
         self.assertIsInstance(result, iris.cube.Cube)
         self.assertArrayAlmostEqual(result.data, expected)
 
+    def test_wind_directions(self):
+        """Test function when a wind direction data cube is provided, and
+        the directions don't cross the 0/360° boundary."""
+        frt_points = [
+            datetime(2015, 11, 19, 0),
+            datetime(2015, 11, 19, 1),
+        ]
+        cube = set_up_variable_cube(
+            np.zeros((2, 2), dtype=np.float32),
+            name="wind_direction",
```

Comment:
This could be `wind_from_direction` to be consistent with the CF conventions: http://cfconventions.org/Data/cf-standard-names/54/build/cf-standard-name-table.html

Reply:
done